tests/testsuite/publish.rs

```diff
@@ -2462,7 +2477,10 @@ fn api_other_error() {
 [PACKAGING] foo v0.0.1 ([ROOT]/foo)
 [PACKAGED] 4 files, [FILE_SIZE]B ([FILE_SIZE]B compressed)
 [UPLOADING] foo v0.0.1 ([ROOT]/foo)
-[ERROR] failed to publish to registry at http://127.0.0.1:[..]/
+[ERROR] failed to publish `foo` v0.0.1
+
+Caused by:
+  failed to publish to registry at http://127.0.0.1:46397/
```

Comment:
Why are we reporting two `failed to publish` messages?  Shouldn't we just put all of this information in the one `failed to publish` message?